Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node
```

Comment:
Specify image which has specific node version rather than taking always the latest. This makes sure that all devs in the team have same config